docs/data/base/getting-started/usage/usage.md

```diff
@@ -101,6 +118,9 @@ Many Base components are implemented with the help of hooks.
 (Visit the [React documentation on hooks](https://reactjs.org/docs/hooks-intro.html) to get up to speed on this concept.)
 
 You can use components or hooks—or a combination thereof—when building custom components.
+
+In general, we recommend that you begin building with the component, and if you find that you are too limited by the customization options available, then consider refactoring your component to use the corresponding hook instead.
+
```

Comment:
Also, if you build a component library that should be further customizable, it's better to use hooks.